### PR TITLE
chore: limit deno_ast features

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by littledivy — COMMENTED** [review]
Can you also rebuild the WASM? :-)

**Comment by bartlomieju** [comment]
> Can you also rebuild the WASM? :-)

I would, but there are no instructions in the README :-)

**Comment by bartlomieju** [comment]
Installing `wasmbuild` and running it results in:
```
thread 'main' panicked at 'remaining data [20, 10, 85, 105, 110, 116, 56, 65, 114, 114, 97, 121]', /Users/ib/.cargo/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-cli-support-0.2.78/src/descriptor.rs:111:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
wasm-bindgen failed
make: *** [build] Error 1
```

**Comment by littledivy** [comment]
```
rustup target add wasm32-unknown-unknown
cargo install wasm-bindgen-cli
deno install --unstable -A -f -n wasmbuild https://raw.githubusercontent.com/denoland/wasmbuild/main/main.ts
brew install binaryen

make release
```

**Comment by littledivy** [comment]
> Installing `wasmbuild` and running it results in:
> 
> ```
> thread 'main' panicked at 'remaining data [20, 10, 85, 105, 110, 116, 56, 65, 114, 114, 97, 121]', /Users/ib/.cargo/registry/src/github.com-1ecc6299db9ec823/wasm-bindgen-cli-support-0.2.78/src/descriptor.rs:111:9
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> wasm-bindgen failed
> make: *** [build] Error 1
> ```

Your `wasm-bindgen-cli` version does not match the version in Cargo.toml

File: README.md [hunk 1]
```diff
@@ -56,3 +56,34 @@ SourceMaps:
...
+brew install binaryen
+```
+
+When opening a PR make sure to rebuild WASM by running:
+
+```
+make release
+```
+
+### Troubleshooting
```
Comment (littledivy) [inline README.md:75]: Nice

**Review by littledivy — APPROVED** [review]

**Pull request merged**